docs/install_guides/troubleshooting-issues.md

```diff
@@ -204,3 +204,18 @@ sudo apt install mokutil
 ```sh
 sudo mokutil --disable-validation
 ```
+
+## <a name="yocto-install-issues"></a>Issues While Creating YOCTO Image with Intel® Distribution of OpenVINO™ Toolkit
+
+### Error while adding "meta-intel" layer
+
+When using the `bitbake-layers add-layer meta-intel` command, the following error might occur:
+```sh
+NOTE: Starting bitbake server...
+ERROR: The following required tools (as specified by HOSTTOOLS) appear to be unavailable in PATH, please install them in order to proceed: chrpath diffstat pzstd zstd
+```
+
+To resolve the issue, install the `chrpath diffstat zstd` tools:
+
+```sh
+sudo apt-get install chrpath diffstat zstd
```

Comment:
It seems the code block is missing a closure here.

Reply:
```suggestion
sudo apt-get install chrpath diffstat zstd
```
```

Reply:
I have applied the correction.